MendelaA/HW7/PT1/3.py

```diff
@@ -0,0 +1,9 @@
+def calculate_character (string:str):
```

Comment:
good job
look at this implementation
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result